templates/configurations/number.interface.hbs

```diff
@@ -6,3 +6,22 @@
   <label>Placeholder</label>
   <input class="form-control" type="text" v-model.trim="placeholder" placeholder="Placeholder text" />
 </div>
+<div class="form-group cleafix">
+    <div class="col-xs-12">
+      <label class="control-label">What numbers should be allowed?</label>
+    </div>
+    <div class="col-xs-12">
+      <div class="radio radio-icon">
+        <input type="radio" id="numbers-all" name="numbers-option" v-bind:value="false" v-model="positiveOnly">
+        <label for="numbers-all">
+          <span class="check"><i class="fa fa-circle"></i></span> <strong>Allow all numbers</strong> (positive and negative integers)
```

Comment:
@squallstar Does "(positive and negative integers)" mean floating points are not allowed?

Reply:
@tonytlwu actually all numbers will be allowed because the iOS keyboard allows you to type both `-` and `.`. What do you suggest we write? Simply keep "Allow all numbers"?

Reply:
Yeh I think the 2 options written as below should do, without additional explanations:

- Allow all numbers
- Positive numbers only